### PR TITLE
proposing:  adding Content section to docs for further reading/understanding

### PR DESCRIPTION
**Pull request by sw-yx**
Hi! came here from the recent JSJabber podcast episode and figured i'd help add it to the docs so future people can learn about it. I like having a "community content" section in my docs so that others can learn more and also contribute their own back.  just a proposal - feel free to accept, modify, or reject...

---

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Compliance

- [x] My change isn't breaking (it doesn't cause existing functionality to change).
- [x] I have read the [CONTRIBUTING](https://github.com/sarahdayan/dinero.js/blob/develop/CONTRIBUTING.md) document.
- [x] My code follows the coding style of this project.
- [x] I have properly formatted my commit messages with [cz-cli](https://github.com/commitizen/cz-cli), or manually, following the [Angular Commit Messages Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [x] I have updated the documentation accordingly, or my changes doesn't require documentation changes.
- [x] I have added tests to cover my changes, or my changes doesn't require new tests.
- [x] All new and existing tests pass.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -221,6 +221,16 @@ This is only a preview of what you can do. Dinero.js has extensive documentation
 
 Pull requests are welcome! Please check the [contributing guidelines][dinero-guidelines] for install instructions and general conventions.
 
+## Content
```
Comment (sarahdayan) [inline README.md]:
```suggestion
## Community
```

File: README.md [hunk 2]
```diff
@@ -221,6 +221,16 @@ This is only a preview of what you can do. Dinero.js has extensive documentation
...
+
+Selected content about Dinero:
```
Comment (sarahdayan) [inline README.md]:
```suggestion
Selected content about Dinero.js:
```

File: README.md [hunk 3]
```diff
@@ -221,6 +221,16 @@ This is only a preview of what you can do. Dinero.js has extensive documentation
...
+
+- [JSJ 351: Dinero.js with Sarah Dayan](https://devchat.tv/js-jabber/jsj-351-dinero-js-with-sarah-dayan/)
+- [Build a Shopping Cart with Vue and Dinero.js](https://frontstuff.io/build-a-shopping-cart-with-vue-and-dinerojs)
+- [How to Handle Monetary Values in JavaScript](https://frontstuff.io/how-to-handle-monetary-values-in-javascript)
+- [Comparison with Numeral.js](https://www.reddit.com/r/javascript/comments/84mhrw/dinerojs_an_immutable_library_to_create_calculate/)
+- Submit your blogpost/tutorial!
```
Comment (sarahdayan) [inline README.md]:
```suggestion
- Submit your own blog post/tutorial!
```

**Comment by sarahdayan** [comment]
Hey @sw-yx, thanks for applying the suggestions. Before merging this, can you please:

* Use Markdown reference links instead of inline
* Finish the PR compliance checklist

Thanks!

**Comment by sw-yx** [comment]
hey Sarah, i tried my best but mucking with git history isn't a strong suit of mine haha. i did a rebase and then resolved all the differences. I think its squashed properly now? but the rebase message looks weird and I'm not sure if I'm supposed to change it somehow.

**Comment by sw-yx** [comment]
ok that was a fun adventure with git rebase - i hope it all fits now!

**Comment by sarahdayan** [comment]
Hope you enjoyed the git fun :) Thanks for this!

**Pull request merged**